components/brave_rewards/browser/database/publisher_info_database_unittest.cc

```diff
@@ -1106,27 +1108,23 @@ TEST_F(PublisherInfoDatabaseTest, Migrationv7tov8_ContributionInfo) {
   CreateMigrationDatabase(&temp_dir, &db_file, 7, 8);
   EXPECT_TRUE(publisher_info_database_->Init());
 
-  ContributionInfo contribution;
-  contribution.probi = "1000000000000000000";
-  contribution.month = static_cast<int>(ledger::ActivityMonth::OCTOBER);
-  contribution.year = 2019;
-  contribution.type = static_cast<int>(ledger::RewardsType::ONE_TIME_TIP);
-  contribution.date = 1570614352;
-  contribution.publisher_key = "3zsistemi.si";
+  const std::string publisher_key = "3zsistemi.si";
```

Comment:
Question here - I recognise this string but is it 'owned' by Brave?

Reply:
that's my domain 😃

Reply:
Whups - was thinking 'integration' test - not unit! Discarding this comment.